package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "@viamrobotics/sdk",
-  "version": "0.0.7",
+  "version": "0.0.8",
```

Comment:
will we have to update the version every push to main?

Reply:
nope, only if we want to publish a new version. the underlying workflow we are using does not publish if we don't bump the version: https://github.com/JS-DevTools/npm-publish